gp-includes/misc.php

```diff
@@ -447,6 +447,30 @@ function gp_is_one_of( $value, $list ) {
 	return in_array( $value, $list, true );
 }
 
+/**
+ * Checks if the passed value has only ASCII characters.
+ *
+ * @since 3.0.0
+ *
+ * @param string $value The value you want to check.
+ * @return bool
+ */
+function gp_is_ascii_string( $value ) {
+	return preg_replace( "/[^\x20-\x7E\p{L}]/", '', $value ) === $value;
+}
+
+/**
+ * Checks if the passed value starts and end with a word character.
+ *
+ * @since 3.0.0
+ *
+ * @param string $value The value you want to check.
+ * @return bool
+ */
+function gp_is_starting_or_ending_with_a_word_character( $value ) {
```

Comment:
Rename to `gp_is_starting_and_ending_with_a_word_character` to fix error message.